docker/start-compose-tests-fullstack.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+. common-sudo-fix.sh
```

Comment:
this seems to be a typo from my tests and comparison with your other changes `. ./common-sudo-fix.sh`